recursive_art.py

```diff
@@ -1,38 +1,35 @@
-""" TODO: Put your header comment here """
+"""
```

Comment:
Good job addressing the comments, but I don't see any evidence of new functionality.. Even if it didn't work, include what you tried.